tkg/clusterclient/awsIngressRules.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package clusterclient
+
+import (
+	capav1beta2 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+)
+
+const (
+	DefaultKappControllerHostPort = 10100
+	DefaultAddonsManagerHostPort  = 9865
+	DefaultFeatureGateHostPort    = 9443
+)
+
+var (
+	awsIngressRules = []capav1beta2.CNIIngressRule{
```

Comment:
Maybe we need to move this to a more generic location in the code. and just import that package. 
someplace like consts or something similar that makes it obvious where to add any new rules required for aws

Reply:
moved everything to the same file and kep everything close together since we only have a couple of rules to add. 
If more rules are needed they maybe done later.

Reply:
moved code to same location as function